MANUAL.md

```diff
@@ -0,0 +1,99 @@
+# Prometheus Genesis Kit Manual 
+The Prometheus Genesis Kit deploys a single-VM installation of
+Prometheus, Grafana, and AlertManager. It's capable of monitoring BOSH
+and Cloudfoundry.
+
+## Requirements
+
+[Node Exporter][1] must be installed on all BOSH VMs you would like
+to monitor. Please consult the README of that repository on how to
+setup the BOSH addon.
+
+
+## Features
+
+## SSL Certificates
+
+* `self-signed-certs` - If you wish to have Genesis generate
+  self-signed certs for you. 
+* `provided-cert` - If you have SSL cert/key to provide, which is
+  grabbed from Vault via path:
+  `$GENESIS_VAULT_PATH/nginx/ssl_certificate:certificate` and
+  `$GENESIS_VAULT_PATH/nginx/ssl_certificate:key`
+
+## Authentication
+
+* `http-auth` - By default, only Grafana supports authentication via
+  the web UI. This feature adds HTTP Basic Authentication to
+  AlertManager and Prometheus with automatically generated
+  credentials.
```

Comment:
Should this really be a feature?  It seems to me that auth is a good thing, especially for something like monitoring.